packages/web/src/components/Grid/_tools.scss

```diff
@@ -0,0 +1,34 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+@use '../../tools/breakpoint';
+
+@function breakpointClass($name, $value) {
+    @if ($value > 0) {
+        @return unquote($name + '--');
+    }
+
+    @return '';
+}
+
+@mixin gutters($gutters, $breakpoints) {
+    @each $breakpointName, $breakpointValue in $breakpoints {
+        @if not map.has-key($gutters, $breakpointName) {
+            @error 'Invalid breakpoint specified! #{$breakpointName} doesn\'t exist. Use one of #{map.keys($breakpoints)}';
+        }
+        @include breakpoint.up($breakpointValue) {
+            gap: map.get($gutters, $breakpointName);
```

Comment:
Have we checked with designers the gutter values can be also used for row gaps?

Reply:
For now we leave same gap for both axis. It will be editable when we add spacing utilities.